diff --git a/Gemfile b/Gemfile
index ba696b3..e9acc79 100644
--- a/Gemfile
+++ b/Gemfile
@@ -8,7 +8,7 @@ redmine_version_major = version_file.match(/MAJOR =/).post_match.match(/\d/)[0].
 
 gem "holidays", "~>1.0.3"
 gem "icalendar"
-gem "nokogiri"
+gem "nokogiri",  "~> 1.6.8"
 gem "open-uri-cached"
 gem "prawn"
 gem 'json'
@@ -25,7 +25,7 @@ group :test do
   gem 'cucumber-rails', require: false
   gem "culerity"
   gem "cucumber"
-  gem "capybara", "~> 1"
+  gem "capybara"
   #gem "faye-websocket"
   gem "poltergeist"
   gem "database_cleaner"
